src/main/java/use_case/login/student_login/StudentLoginInteractor.java

```diff
@@ -20,25 +20,30 @@ public StudentLoginInteractor(StudentLoginDataAccessInterface studentDataAccessI
      * @param studentLoginInputData the input data
      */
     public void execute(StudentLoginInputData studentLoginInputData) {
-        final String username = studentLoginInputData.getUsername();
+        final String studentEmail = studentLoginInputData.getStudentEmail();
         final String password = studentLoginInputData.getPassword();
-        if (!studentDataAccessObject.existsByName(username)) {
-            studentLoginPresenter.prepareFailView(username + ": Account does not exist.");
+        if (!studentDataAccessObject.existsByEmail(studentEmail)) {
+            System.out.println(studentEmail);
+            System.out.println(studentDataAccessObject.existsByEmail(studentEmail));
```

Comment:
These lines are printed to the console. Are these here for testing purposes? They should be removed if so.

Reply:
Yes they were. Thank you for spotting that!